.gitmodules

```diff
@@ -16,5 +16,5 @@
 	url = https://github.com/aws-quickstart/quickstart-aws-vpc.git
 [submodule "submodules/quickstart-linux-bastion"]
     path = submodules/quickstart-linux-bastion
-    branch = main
+    branch = develop
```

Comment:
We prefer pinning to the main branch here; We've migrated away from a `develop` branch strategy

Reply:
got it; easy change ✅